ios/gutenberg/GutenbergViewController.swift

```diff
@@ -194,6 +195,10 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
     func gutenbergDidLogUserEvent(_ event: GutenbergUserEvent) {
         print("Gutenberg loged user event")
     }
+
+    func gutenbergDidRequestMention(callback: @escaping (Result<String, NSError>) -> Void) {
+        callback(.success("matt"))
```

Comment:
I'd be in favor of "sergio" here for balance, but I guess the name "matt" get's special treatment around here ;-P